### ISSUE TITLE
Share accessibility plugins to add

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#84]
We have 2 locations where we mention plugins.
What are trusted plugins to add to those lists, please comment withthis issue when you have an addition?

1: On [Start with accessibility](https://wpaccessibility.org/docs/start/) with the (yet to be merged) page WP Plugins
2: On [Testing](https://wpaccessibility.org/docs/testing/)  with the (yet to be merged) page WP plugins for testing

### Editing Tools

- [Accessibility New Window Warnings](https://wordpress.org/plugins/accessibility-new-window-warnings/) – Automatically adds visible warnings on all links that open in new tabs or windows
- [Screen Reader Text Format](https://wordpress.org/plugins/screen-reader-text-format/) – Adds a block editor text control to set content as screen reader only.
- [Lang attribute for the Block Editor](https://wordpress.org/plugins/lang-attribute/) – Enables users to set the language of a portion of content in the editor.
- [Able Player for WordPress](https://wordpress.org/plugins/ableplayer/) – Able Player is a media player that offers extensive support for accessible features such as Audio Description, speed adjustment, and interactive transcriptions.

### Plugin or Theme Specific Patterns:

- [Contact Form 7: Accessible defaults](https://wordpress.org/plugins/contact-form-7-accessible-defaults/) – generates accessible contact form defaults in Contact Form 7.
- [Accessible Elementor popups](https://wordpress.org/plugins/accessible-elementor-popups-by-accessibility-zone/) – Adds accessibility support to popups in Elementor.

### General Purpose:

[WP Accessibility](https://wordpress.org/extend/plugins/wp-accessibility/) — provides features that fix accessibility issues in your site.

### Testing Tools & Content Analysis:

- [Accessibility Checker](https://wordpress.org/plugins/accessibility-checker/) – Accessibility testing tool that runs directly in your WordPress site.
- [WP ADA Compliance Check Basic](https://wordpress.org/plugins/wp-ada-compliance-check-basic/) – Accessibility testing tool that runs directly in your WordPress site. (Limit 15 posts or pages)
- [Fix Alt Text](https://wordpress.org/plugins/fix-alt-text/) – Helps user find images with missing alternative text.
- [WP Accessibility Tools & Missing Alt Text Finder](https://wordpress.org/plugins/tool-for-ada-section-508-and-seo/) – Provides accessibility testing and media analysis support.
- [Editoria11y](https://wordpress.org/plugins/editoria11y-accessibility-checker/) – Accessibility checking tool geared towards content authors.
- [Sa11y](https://wordpress.org/plugins/sa11y/) – Accessibility checking tool geared towards content authors.
